stjude/proteinpaint
# Notify browser of dev events, including live-reload

## Description

... to help minimize confusion of when bundling has completed or not.

TODO: also trigger `tsc` type check and `GDC API IP address` notifications.

To test:
1. Open any pp url, a green notification should pop up.
![Screenshot 2024-07-10 at 1 23 06 PM](https://github.com/stjude/proteinpaint/assets/411031/1cb7716e-89fa-4fb4-b29a-ceb6f448dd8e)

2. Then select a client-side code to edit with a syntax error, like a randomly placed `if` keyword, then save. The browser should show a red notification. 

![Screenshot 2024-07-10 at 1 24 53 PM](https://github.com/stjude/proteinpaint/assets/411031/f3154a4f-3b39-4e03-87c2-731d3ecede66)

3. Undo the syntax error, and the green notification should show up again and automatically trigger a browser refresh/reload.

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
